shared/js/background.js

```diff
@@ -200,8 +201,14 @@ chrome.webRequest.onBeforeRequest.addListener(
                     console.info( "blocked " + utils.extractHostFromURL(thisTab.url)
                                  + " [" + tracker.parentCompany + "] " + requestData.url);
 
+                    // return surrogate redirect if match, otherwise
                     // tell Chrome to cancel this webrequest
-                    return {cancel: true};
+                    if (tracker.redirectUrl) {
+                        console.info("serving surrogate content for: ", tracker.url)
```

Comment:
Can you remove this message? It's the same as the one coming`checkSurrogateList` in trackers.js

Reply:
👍  done